app/controllers/api_controller/reports.rb

```diff
@@ -1,12 +1,20 @@
 class ApiController
   module Reports
+    SCHEDULE_ATTR = %w(start_date interval time_zone send_email).freeze
```

Comment:
What's special about these attrs? Are they black/white listed? Could this constant be given a more descriptive name?